lib/query-interface.js

```diff
@@ -747,7 +747,7 @@ module.exports = (function() {
     }
 
     var sql = this.QueryGenerator.selectQuery(tableName, options, Model)
-      , queryOptions = Utils._.extend({ transaction: options.transaction }, { plain: true, raw: true, type: QueryTypes.SELECT })
+      , queryOptions = Utils._.extend({ transaction: options.transaction, plain: options.plain!==false }, { raw: true, type: QueryTypes.SELECT })
```

Comment:
This should probably be changed to `plain: options.plain` and then have `options.plan = options.plain === undefined ? true : options.plain`